test/CheckRightsService.Business.UnitTests/CheckRightsService.Business.UnitTests.csproj

```diff
@@ -1,15 +1,26 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
 
     <IsPackable>false</IsPackable>
+
+    <AssemblyName>LT.DigitalOffice.CheckRightsService.Business.UnitTests</AssemblyName>
+
+    <RootNamespace>LT.DigitalOffice.CheckRightsService.Business.UnitTests</RootNamespace>
```

Comment:
Remove empty line and do also in others project

Reply:
Fixed.